nequip/train/trainer.py

```diff
@@ -1002,10 +1002,10 @@ def save_ema_model(self, path, blocking: bool = True):
 
     def save_model(self, path, blocking: bool = True):
         with atomic_write(path, blocking=blocking, binary=True) as write_to:
-            if isinstance(self.model, torch.jit.ScriptModule):
-                torch.jit.save(self.model, write_to)
-            else:
-                torch.save(self.model.state_dict(), write_to)
+            # if isinstance(self.model, torch.jit.ScriptModule):
+            #     torch.jit.save(self.model, write_to)
+            # else:
+            torch.save(self.model.state_dict(), write_to)
```

Comment:
@Linux-cpp-lisp  is this correct? if it is right, we can delete the commented block.

Reply:
msg'd on slack